code/src/components/Item.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { todos } from 'reducers/todos'
+
+export const Item = (props) => {
+  const dispatch = useDispatch()
+
+  const handleCheckbox = () => {
+    dispatch(todos.actions.toggleIfChecked(props.item.id))
+  }
+
+  const handleRemove = () => {
+    dispatch(todos.actions.removeItem(props.item.id))
+  }
+
+  return (
+    <li>
+      {props.item.name}
+      
+        <input type="checkbox" id={props.item.id} name="do" checked={props.item.ifChecked} onChange={handleCheckbox} />
+      <label for={props.item.id}></label>
+
```

Comment:
To avoid having to write `props.item.name props.item.id` and so on you could destructure this in the beginning of the component.
Either by:
`const {name, id, ifChecked} = props.item`
or 
`export const Item = ({name, id, ifChecked}) => { ... `